src/styles/isomer-cms/pages/Editor.module.scss

```diff
@@ -1,5 +1,11 @@
+@use "sass:meta";
+
 @import "../elements/variable";
 
+.preview-styles {
+  @include meta.load-css("../../preview-panel.scss");
```

Comment:
from the docs located [here](https://sass-lang.com/documentation/modules/meta), this means that the css under `preview-panel.scss` would be scoped under `.preview-styles` right?